x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phase_footer/phase_footer.tsx

```diff
@@ -34,10 +34,10 @@ export const PhaseFooter: FunctionComponent<Props> = ({ phase }) => {
 
   const phaseDescription = isDeletePhaseEnabled
     ? i18n.translate('xpack.indexLifecycleMgmt.editPolicy.phaseTiming.beforeDeleteDescription', {
-        defaultMessage: 'Data will be deleted after this phase',
+        defaultMessage: 'Policy deletes data after this phase',
       })
     : i18n.translate('xpack.indexLifecycleMgmt.editPolicy.phaseTiming.foreverTimingDescription', {
-        defaultMessage: 'Data will remain in this phase forever',
+        defaultMessage: 'Data remains in this phase forever',
```

Comment:
I'd stick with the imperative:

```suggestion
        defaultMessage: 'Keep data in this phase forever',
```